src/box/box.cc

```diff
@@ -4011,6 +4011,19 @@ box_upsert(uint32_t space_id, uint32_t index_id, const char *tuple,
 	return box_process1(&request, result);
 }
 
+API_EXPORT int
+box_insert_arrow(uint32_t space_id, struct ArrowArray *array,
```

Comment:
Please squash the last two patches. Keeping these changes apart make it difficult to review. The new request doesn't make much senses without the new function IMO.

Reply:
Done.